src/site_upload/cache_api/cache_api.py

```diff
@@ -57,7 +57,7 @@ def cache_api_data(s3_client, s3_bucket_name: str, db: str, target: str) -> None
                 "id": f"{dp_detail['study']}__{dp_detail['name']}__{version}",
             }
             if "__flat" in dp_dict["s3_path"]:  # pragma: no cover
-                site = dp_dict["s3_path"].split("/")[5]
+                site = dp_dict["s3_path"].split("/")[2]
```

Comment:
![image](https://github.com/user-attachments/assets/67b31323-57aa-428a-8974-e57bcd39e8a6)

Reply:
it's on my list, i know you hate it